.github/workflows/claude.yml

```
@@ -1,34 +1,29 @@
-name: Claude Code (Enhanced)
+name: Claude Code
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)